sdk/keyvault/keyvault-keys/package.json

```diff
@@ -19,7 +19,7 @@
   },
   "main": "./dist/index.js",
   "module": "dist-esm/src/index.js",
-  "types": "./types/src/index.d.ts",
+  "types": "./types/keyvault-keys.d.ts",
```

Comment:
Shouldnt we update the `files` section to include just the above file instead of `types/lib/`?